app/stapf/helpers.py

```diff
@@ -29,9 +29,9 @@ def send_batch_mails(batch):
                     msg.attach(batch.decision.filename, "application/pdf", fp.read())
                 conn.send(msg)
 
-        msg = Message(batch.subject, recipients=[current_app.config['MAIL_STAPF']],
```

Comment:
Why is this being commented out? Could this be hidden behind a config variable or a check if `MAIL_STAPF` is even set?